libraries/timeline-view-compose/src/main/java/com/trendyol/uicomponents/timelineviewcompose/model/FakeTimelineItemProvider.kt

```diff
@@ -0,0 +1,54 @@
+package com.trendyol.uicomponents.timelineviewcompose.model
+
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.font.FontFamily
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.unit.Dp
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+
+object FakeTimelineItemProvider {
+
+    internal fun provideTimelineItem(
```

Comment:
I think these function shouldn't be part of the library.

Reply:
I've enabled proguard and delete this